rails/app/javascript/components/App.jsx

```diff
@@ -157,6 +171,23 @@ class App extends Component {
         });
         break;
       }
+      case FILTER_CATEGORIES[3]: {
+        // second category: topic
+        filteredStories = this.props.stories.filter(story => {
+          if (
+            story.points.some(point => {
+              return (
```

Comment:
@julinvictus you fixed the category dropdown, but the filter now has the same problem:

this filter just wants to be "if i am filtering by topic, return me all stories that meet this filter

and so theres no need to iterate through points, since we're just looking at the top level story'

something akin to
  filteredStories = this.props.stories.filter(story => {
          if(story.topic) {
            return story.topic.toLowerCase() == item.toLowerCase();
          }
        });